### PR TITLE
I think i've done it?

### PR DESCRIPTION
**Pull request by AlexandruBuhai**

### CODE REVIEW

File: pages/event.js [hunk 1]
```diff
@@ -19,7 +18,7 @@ export default class Event extends React.Component {
         return <div>
             <h1>Event example</h1>
             <Navigation/>
-            Type your text: <input type="text" value={this.state.value} onChange={handleChange}/>
+            Type your text: <input type="text" value={this.state.value} onChange={handleChange.bind(this)}/>
```
Comment (andrasfodor) [inline pages/event.js]: :+1:

File: pages/index.js [hunk 2]
```diff
@@ -6,14 +6,17 @@ export default class Index extends React.Component {
     constructor() {
         super();
 
+        this.getCounterValue = this.getCounterValue.bind(this);
```
Comment (andrasfodor) [inline pages/index.js]: :+1: 
or this.getCounterValue as arrow function

File: pages/index.js [hunk 3]
```diff
@@ -24,7 +27,7 @@ export default class Index extends React.Component {
     /**
      * Increase counter
      */
-    inceaseCounter() {
+    increaseCounter = () => {
```
Comment (andrasfodor) [inline pages/index.js]: :+1: 
Or bind it in constructor

**Comment by AlexandruBuhai** [comment]
Thanks for the review!

**Comment by andrasfodor** [comment]
live:andrasfodor_2 - please add me to your skype contact list

**Pull request closed without merging**